src/components/TalentPage/TalentPageLayout.tsx

```diff
@@ -0,0 +1,238 @@
+								<span id="save"> {`${t('TalentCompanyPage.saved')}`}</span>
+							</Button>
+						</ButtonBlock>
+						{active?.discover === 'discover' && (
+							<>
+								<Title className="filterTitle">
+									{`${t('TalentCompanyPage.filter')}`}{' '}
+									<span>
+										<BsArrowRightCircle className="arrowRight w-40 h-40" onClick={getFilterList} />
+									</span>
+								</Title>
+								<Controller
+									name="category"
+									control={control}
+									render={({ field }) => {
+										return (
+											<Select
+												{...field}
+												options={selection}
+												placeholder="Categories"
+												onChange={select => select && setSelect({ name: select.label })}
+												theme={theme => ({
+													...theme,
+													borderRadius: 0,
+													colors: {
+														...theme.colors,
+														neutral50: 'black', // Placeholder color
+													},
+												})}
+												className={`${errors.category ? 'is-invalid' : ''}`}
+											/>
+										);
+									}}
+								/>
+								<BorderStyled className="borderResults">{`${t(
+									'TalentCompanyPage.results',
+								)}`}</BorderStyled>
+							</>
+						)}
+					</WrapperSidePanel>
+				)}
+				{(showFilterList || matchesQuery) && active?.discover === 'discover' && (
+					<Wrapper>
+						<div>
+							<Title>
+								<span>
+									<BsArrowLeftCircle className="arrowRight w-40 h-40" onClick={getFilterList} />
+								</span>
+								{`${t('TalentCompanyPage.title')}`}
+							</Title>
+						</div>
+						<InputContainer className="input-group rounded">
+							<IconSearch>
+								<ImSearch />
+							</IconSearch>
+							<Input
+								type="search"
+								onChange={(event: { target: { value: React.SetStateAction<string> } }) =>
+									setSearch(event.target.value)
+								}
+								placeholder="Discover"
+								value={search}
+								aria-label="Discover"
+								aria-describedby="search-addon"
+							/>
+						</InputContainer>
+						<BorderStyled>{`${t('TalentCompanyPage.skill')}`}</BorderStyled>
+						<SkillsButtonsBlock
+							data-toggle="buttons"
+							className="btn-group btn-group-toggle flex-wrap"
+						>
+							{optionButtons}
+						</SkillsButtonsBlock>
+					</Wrapper>
+				)}
+			</MainBlockWrapper>
+			{active?.discover === 'discover' && data?.profile && !isLoading ? (
+				<PaginationBlock>
+					<ProfileBlock>
+						{data?.profile &&
+							data?.profile.map((item: Filter, index: React.Key | null | undefined) => {
+								return (
+									<div key={index}>
+										<FilterProfileUser item={item} />
+									</div>
+								);
+							})}
+					</ProfileBlock>
+					<Pagination filterPerPage={data.limit} total={data.total} paginate={paginate} />
+				</PaginationBlock>
+			) : (
+				<Suspense fallback={<div>{`${t('PostDetailPage.loading')}`}</div>}></Suspense>
+			)}
+			{active?.hires === 'hires' && <MyHiresCompany />}
```

Comment:
condition only with constants, never with strings, its bad practice

Reply:
fixed it